config/config.yaml

```diff
@@ -4,6 +4,8 @@ kraken:
         -    scenarios/etcd.yml
         -    scenarios/openshift-kube-apiserver.yml                           
         -    scenarios/openshift-apiserver.yml
+    node_scenarios:                                        # List of chaos node scenarios to load
+        -   scenarios/node_scenarios_example.yml
```

Comment:
It might be also nice to show an example of the node_crash and stop_kubelet in a separate scenario or also in this one

Reply:
In addition, it could be nice to maybe have a separate README defining how to set up a node scenario and all the possible actions that you can have.

Reply:
I have made changes in the README, PTAL.

Reply:
LGTM thanks for adding that!